README.md

```diff
@@ -87,6 +87,26 @@ cadenzaClient.show('{embeddingTargetId}', {
 });
 ```
 
+#### Show an Embedding Target in Simplified Operation Mode
+
+Workbook embedding targets can be shown with simplified operation mode enabled by setting the "operationMode" option to "simplified".
+
+```javascript
+cadenzaClient.show('{embeddingTargetId}', {
+  operationMode: 'simplified'
+});
+```
+
+#### Show an Embedding Target with Disabled Features
+
+Embedding targets can be shown with certain UI features disabled. To disable the designer, add "workbook-design" to the "disabledUiFeatures" option. To disable editing of workbook layout and design, add "workbook-view-management".
+
+```javascript
+cadenzaClient.show('{embeddingTargetId}', {
+  disabledUiFeatures: ['workbook-design', 'workbook-view-management']
+});
+```
+
```

Comment:
To keep the list of examples concise, maybe it would be enough to add these params to the existing "Show an embedding target" example?

Reply:
How's that look?

It didn't make sense to include 'workbook-view-management' in the example, because it has no effect in simplified operation mode, and nobody is going to specify "operationMode: 'normal'" since that's the default anyway.